view/src/extras/alert.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Snackbar from "@material-ui/core/Snackbar";
+import MuiAlert from "@material-ui/lab/Alert";
+import { makeStyles } from "@material-ui/core/styles";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    "& > * + *": {
+      marginTop: theme.spacing(2),
+    },
+  },
+}));
+
+/**
+ * Creates an alert that shows up on the snackbar to help
+ * users visualize when certain action are completed or if
+ * errors occur
+ * @param props Determine severity of alert, message presented
+ *              and control opening and closing of the alert
+ */
+export default function CustomizedSnackbars(props) {
+  // Styling and duration of the alert
+  const classes = useStyles();
+  const alertDuration = 6000;
```

Comment:
Seems like this should be in props too, don't you think?  Maybe use this as a default if it's not specified there.